### ISSUE TITLE
Automatic download and execution of browser from Google should be noted in README

### ISSUE THREAD

**Issue by thingsiplay** [charmbracelet/vhs#438]
I just installed vhs through my package manager. And when I run the application I was shocked, as it downloads an executable from the web without disclosing it would. I think it is very important to note this in the README it would connect to Google servers and download entire browser to operate correctly. After my research I understand why this is happening, as it is done through a library or application vhs depends on. I think this is important for the user to know before using the program. Please add this as a note to the README.

**Comment by pspiagicw** [comment]
Also, it running Chrome requires installation of libraries like (libatk-1-0 etc) which power google-chrome. 
This makes the dependencies of `vhs` close to 300+MB on a Ubuntu Server.

It should be mentioned in the README that Chrome would be downloaded and all the dependencies of Chrome should be present.

**Comment by benstigsen** [comment]
I second this.
Was thoroughly confused when I ran `vhs foo.tape` and a browser was downloaded.

**Comment by TWinsnes** [comment]
I'd really appreciate a message with a confirmation before auto-installing a browser on top of the update to the readme. It was an unpleasant surprise

**Comment by TWinsnes** [comment]
For those looking for the install location, it's managed by the go-rod package and on osx it's installed to `~/.cache/rod/browser/`

**Comment by SuicideShow** [comment]
Yes can only confirm that this was very unexpected and unwanted to see suddenly a google browser being installed.
I really hope it would be possible without it :/

**Comment by xxxserxxx** [comment]
> I've worked around this, but using it in a docker.

Why would this work? Won't vhs just download chromium in the container instead? I mean, if vhs isn't downloading it in the container, the container itself already contains chromium? You're still incurring a massive 300MB or whatever download; now it's just hidden behind docker. Or, is there some magic where vhs in a container doesn't need chromium -- in which case, it shouldn't need chromium _outside_ of a container.

**Comment by dimk90** [comment]
For those hoping this is possible without a browser: `tmux` + [`asciinema`](https://github.com/asciinema/asciinema) + [`agg`](https://github.com/asciinema/agg) needs no Chromium at all. [S-VHS](https://github.com/dimk90/s-vhs) wraps those three into tape-like recordings, nothing gets downloaded behind your back and the recording stays a single reproducible script, like a `.tape`.
